app/src/main/java/org/wikipedia/readinglist/ReadingListsFragment.kt

```diff
@@ -842,6 +862,43 @@ class ReadingListsFragment : Fragment(), SortReadingListsDialog.Callback, Readin
         }
     }
 
+    private fun setupRecommendedReadingListDiscoverCardView(recommendedArticles: List<RecommendedPage>) {
+        binding.discoverCardView.isVisible = true
+        binding.discoverCardView.setContent {
+            var images by remember { mutableStateOf(recommendedArticles.mapNotNull { it.thumbUrl }) }
+            var isNewListGenerated by remember { mutableStateOf(Prefs.isNewRecommendedReadingListGenerated) }
+            val subtitle = when (AccountUtil.isLoggedIn) {
+                true -> { getString(R.string.recommended_reading_list_page_subtitle_made_for,
+                    AccountUtil.userName) }
+                false -> { getString(R.string.recommended_reading_list_page_logged_out_subtitle_made_for_you) }
+            }
+
+            val description = when (Prefs.recommendedReadingListUpdateFrequency) {
+                RecommendedReadingListUpdateFrequency.DAILY -> R.string.recommended_reading_list_page_description_daily
+                RecommendedReadingListUpdateFrequency.WEEKLY -> R.string.recommended_reading_list_page_description_weekly
+                RecommendedReadingListUpdateFrequency.MONTHLY -> R.string.recommended_reading_list_page_description_monthly
+            }
+            BaseTheme {
+                RecommendedReadingListDiscoverCardView(
+                    modifier = Modifier
+                        .clickable {
+                           // @TODO: open recommended list screen
+                            isNewListGenerated = false
+                            Prefs.isNewRecommendedReadingListGenerated = false
```

Comment:
Instead of having a new preference, can we reuse the value below?
```
val recommendedArticles = RecommendedReadingListViewModel.getNewRecommendedArticles()
```